src/wp2hugo/internal/hugogenerator/hugo_gen_setup.go

```
@@ -419,6 +419,10 @@ func (g Generator) downloadPageMedia(outputMediaDirPath string, p *hugopage.Page
 		for _, prefix := range prefixes {
 			link = strings.TrimPrefix(link, prefix)
 		}
+		url, _ := url.Parse(link)
```

Please don't discard errors

```
url, err := url.Parse(link)
```